corehq/apps/reports_core/filters.py

```diff
@@ -197,7 +197,7 @@ def __init__(self, name, required=True, label='Choice List Filter',
         self.choices = choices or []
 
     def value(self, **kwargs):
-        choice = kwargs[self.name]
+        choice = unicode(kwargs[self.name])
```

Comment:
Was waiting for tests, which have now passed, so I'm happy to merge unless you want me to hold off.